packages/sst/src/runtime/handlers/rust.ts

```diff
@@ -78,6 +78,7 @@ export const useRustHandler = Context.memo(async () => {
             path.join(input.out, "handler")
           );
         } catch (ex) {
+          console.log(ex);
           throw new VisibleError("Failed to build");
```

Comment:
@gvidon instead of `console.log`, can you try this in both places? And see if errors details show up.

```diff
- throw new VisibleError("Failed to build");
+ return {
+   type: "error",
+   errors: [String(ex)],
+ };
```

^ for context. This is what we do for other runtimes, not sure why we r throwing here. @thdxr might know better.

Reply:
@fwang thanks, replaced in PR.